source/github-helpers/pr-branches.ts

```diff
@@ -76,3 +77,42 @@ export function getBranches(): {base: PrReference; head: PrReference} {
 		},
 	};
 }
+
+export function getFilenames(menuItem: HTMLElement): {original: string; new: string} {
+	if (menuItem.tagName === 'A') {
+		const fileUrl = menuItem
+			.parentElement!
+			.parentElement!
+			.querySelector('li[data-variant="danger"] a')!
```

Comment:
Note that .closest() only replaces .parentElement.parentElement, not querySelector. So you're most likely looking for `.closest('ul').querySelector('li[data-variant="danger"] a')`

Also I think that you can't delete a deleted file, so this element won't exist on deleted files, so `restore-file` cannot restore deleted files. You might need to get the URL from `View file` instead. You can `querySelector` its `svg` to make it easier

Reply:
Nice thanks, `li:has(svg.octicon-eye)` did the trick